dts/bindings/mtd/nordic,qspi-nor.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Any QSPI NOR flash that supports the JEDEC CFI interface.
```

Comment:
This will show up in the header as a comment above the generated macros, so might be better to make it "QSPI NOR flash supporting the JEDEC CFI interface".